pkg/ovs/ovsconfig/ovs_client_test.go

```diff
@@ -0,0 +1,46 @@
+// Copyright 2020 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package ovsconfig
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+type A struct {
+	string
+}
+
+func TestOVSClient(t *testing.T) {
+
+	// We expect this test to fail by panicking in cases where ovs version doesnt return the right values.
```

Comment:
comment inaccurate, there is no panic involved as far as I can tell, `ParseOvsVersion` just returns an error and we validate the error. If the result is not expected, the test fails (but once again no panic).

Reply:
fixing